tests/adapter_tests_async/test_async_asgi.py

```diff
@@ -221,4 +221,4 @@ async def test_url_verification(self):
 
         assert response.status_code == 200
         assert response.headers.get("content-type") == "application/json;charset=utf-8"
-        assert_auth_test_count(self, 1)
+        assert_auth_test_count(self, 0)
```

Comment:
Indeed, this should be zero, but perhaps this had been wrong due to the current test runner mechanism's behavior, right? Nice improvement 👍

Reply:
Yess, I suspect this may be caused by the current test runner mechanism's behavior but I need to dig into each individual test to see what the expected behavior should be and why these values are different, there seem to be a few like these 🤔